src/main/java/de/tum/in/www1/artemis/repository/ComplaintResponseRepository.java

```diff
@@ -46,15 +46,14 @@
     @Query("""
             SELECT COUNT (DISTINCT p) FROM StudentParticipation p
             WHERE p.exercise.id = :#{#exerciseId}
+            AND (p.testRun = FALSE OR p.testRun IS NULL)
```

Comment:
```suggestion
            AND p.testRun = FALSE
```

Reply:
the value cannot be null, and OR can be significantly slower in such cases

Reply:
fixed in 139c8fb547deadc2b10c0e497d91389dcb5f736c